src/github.com/getlantern/lantern-ui/app/index.html

```diff
@@ -57,10 +57,10 @@
   </head>
 <!--[if lt IE 10]>
   <body>
-          <p class="chromeframe">You are using an outdated browser. Please <a href="https://www.google.com/chrome/">upgrade your browser</a> or <a href="https://www.google.com/chromeframe/?redirect=true">install Google Chrome Frame</a> and try again.</p>
+          <p class="chromeframe">You are using an outdated browser. Please <a href="https://www.google.com/chrome/">upgrade your browser</a> and try again.</p>
   </body>
 <![endif]-->
-<!--[if gte IE 10]>
+<!--[if gte IE 10]><!-->
```

Comment:
Ref http://www.sitepoint.com/web-foundations/internet-explorer-conditional-comments/ for explanations.